model/variable.h

```diff
@@ -76,6 +76,7 @@ namespace minsky
   private:
     CLASSDESC_ACCESS(VariableBase);
     std::string m_name; 
+    std::pair<std::string,std::string> m_dimLabelsPicked=std::make_pair("","");    
```

Comment:
You could just use the default initializer for pair here.

Reply:
Wasn't sure. Also fixed that up now.